### ISSUE TITLE
Setup the local stress client and perform periodic prolonged runs.

### ISSUE THREAD

**Issue by jsquire** [Azure/azure-sdk-for-net#8573]
# Summary

To ensure that the track two client library performs consistently over an extended period of time and does not exhibit instability or excess resource consumption, a stress harness was created to allow running core scenarios repeatedly for an extended period of time.  The focus of this testing to validate data integrity and stability over time rather than raw performance.

# Scope of Work

- Update the stress harness to work with the reworked client hierarchy.

- Setup and configure the stress client to perform runs when manually requested.

- Perform the initial run, allowing the scenarios to loop for at least 3 days.

- Review the results and create any work items for issues found.

- Scenarios should be self-contained and execute in parallel, to a reasonable degree.

# Out of Scope

- Collection of performance-based metrics.  

- Advanced diagnostics, inspection, or reporting; this is a minimal harness and not intended to be an industrial-strength product.

# Success Criteria

- The refactoring and enhancements detailed by the scope have been completed.

- The harness can be executed and has proven stable over a period of a working day such that we have confidence any problems are the result of our Event Hubs client library and not the harness itself.

- An initial run of testing has been performed; issues have been opened for any issues discovered.
